src/open_inwoner/openzaak/tests/test_cases.py

```diff
@@ -340,13 +337,19 @@ def test_list_open_cases(self, m):
                     "description": self.zaaktype["omschrijving"],
                     "current_status": self.status_type1["omschrijving"],
                 },
+                {
+                    "uuid": self.zaak3["uuid"],
```

Comment:
Can we give the (numbered) test resources a more meaningful name so the test stay readable: `self.zaak1`, `self.zaak3` don't mean a lot. 

Ideally something short that shows the intent of the resource (eg: `self.zaak_intern` is an internal case)

Ex: `status_type1` should be `status_type_initial`, and `status_type2` like  `status_type_final`

Reply:
This needs to be coordinated across all tests that make use of this data. In this context we could refactor the relevant tests to make the use of zaken data uniform. I took a stab at encapsulating the zaken test data in a mixin as part of [PR 815](https://github.com/maykinmedia/open-inwoner/pull/815). Would be happy to take this refactor further and rename the individual zaken, as you suggested. Let's discuss this.